docs/fdc3-glossary.md

```diff
@@ -27,6 +27,9 @@ For the purposes of this Standard, the following definitions apply. Other terms
 - **DAB**: Acronym of Desktop Agent Bridge.
 - **Desktop Agent**: A desktop component (or aggregate of components) that serves as a launcher and message router (broker) for applications in its domain. The primary interface of the FDC3 API.
 - **Desktop Agent Bridge**: An independent service that Desktop Agents connect to which allows them to relay requests to other Desktop Agents also connected to the bridge, allowing FDC3-based interop to extend across multiple Desktop Agents and machines.
+- **Desktop Agent Communication Protocol**: JSON communication protocol for the Desktop Agent API. Used by a Desktop Agent Proxy interface over the Channel Messaging API (as specified by the Web Connection Protocol) to communicate with a Desktop Agent running in another window/frame.
```

Comment:
Would be good to add the bridging protocol in here too - reading this it's not clear which you're referring to.

Also I think we need acronyms for these protocols.. that would really help

Reply:
I can add the acronyms and a name for the Bridging protocol. Good suggestion